### DISCUSSION TITLE
sass/css Vite equivalent of font-url

### DISCUSSION THREAD

**Discussion by judocode** [ElMassimo/vite_ruby#142] [Q&A] [answered]
Greetings!

I'm currently converting from webpacker to Vite and I am loving it so far! I am getting hung up on one portion of it which is finding what the equivalent to referencing a font with `font-url` in a sass file is. Here is my current setup:

Currently, I have fonts in the `app/assets/fonts` folder, such as:
```
app
|
-- assets
      |
      -- fonts
           |
           -- graphik
                |
                -- Graphik-Medium-Web.eot
```

And in my `app/assets/stylesheets/application.scss` file, I can reference the font with: `font-url('graphik/Graphik-Medium-Web.woff2')`.

However, this does not work when I am using Vite. I see a request made for the font in the network tab as `http://localhost:8080/assets/graphik/Graphik-Medium-Web-[digest].woff2`, but the reference in the `application.scss` in the network tab is still the original `ffont-url('graphik/Graphik-Medium-Web.woff2')`.

I have tried to add the assets paths in `config/vite.json` by adding:
```
"watchAdditionalPaths": [
      "app/assets/**/*"
    ]
```

But that didn't seem to help either.

Am I missing some sort of configuration setting to get this all hooked up with vite_ruby?

Thanks!

**Comment by ElMassimo** [comment, marked as answer]
Answered in https://github.com/ElMassimo/vite_ruby/discussions/144#discussioncomment-1710394.

**Reply by judocode** [reply]
@ElMassimo So sorry Github was down when I posted this and said it failed but apparently it did not. I don't see an option to delete, but you can feel free and delete this one if you are able to!
